docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -423,33 +534,36 @@
     "    google_service_context,\n",
     ")\n",
     "from llama_index import ServiceContext, VectorStoreIndex\n",
-    "from llama_index.llms import PaLM\n",
     "from llama_index.postprocessor import LLMRerank\n",
     "from llama_index.query_engine import RetrieverQueryEngine\n",
     "from llama_index.retrievers import VectorIndexRetriever\n",
     "\n",
     "# Set up the query engine with a reranker.\n",
-    "store = GoogleVectorStore.from_corpus(corpus_id=\"some-corpus-id\")\n",
+    "store = GoogleVectorStore.from_corpus(corpus_id=corpus_id(1))\n",
     "index = VectorStoreIndex.from_vector_store(\n",
-    "    vector_store=store, service_context=google_service_context\n",
+    "    vector_store=store,\n",
+    "    service_context=google_service_context,\n",
     ")\n",
     "response_synthesizer = GoogleTextSynthesizer.from_defaults(\n",
-    "    temperature=0.7, answer_style=GenerateAnswerRequest.AnswerStyle.ABSTRACTIVE\n",
+    "    temperature=0.2,\n",
+    "    answer_style=GenerateAnswerRequest.AnswerStyle.ABSTRACTIVE,\n",
     ")\n",
     "reranker = LLMRerank(\n",
-    "    top_n=10, service_context=ServiceContext.from_defaults(llm=PaLM())\n",
+    "    top_n=10,\n",
+    "    service_context=ServiceContext.from_defaults(llm=gemini),\n",
```

Comment:
need to add `embed_model="local"` otherwise there will be errors for OpenAI key missing

Reply:
Good catch! Used embed_model=None so that OpenAI key isn't needed.